src/meta/src/manager/catalog/mod.rs

```diff
@@ -2193,6 +2194,47 @@ impl CatalogManager {
         Ok(())
     }
 
+    pub async fn comment_on(&self, comment: Comment) -> MetaResult<NotificationVersion> {
+        let core = &mut *self.core.lock().await;
+        let database_core = &mut core.database;
+        database_core.ensure_table_view_or_source_id(&comment.table_id)?;
```

Comment:
```suggestion
        database_core.ensure_table_id(&comment.table_id)?;
```
Or you wanna support other relations as well in the future?

Reply:
Yes, other relations should be supported in the future, but for now it is indeed possible to only ensure table.